impl/client/src/main/javascript/web/dojo/dijit/themes/pentaho/PageControl.css

```diff
@@ -43,11 +47,11 @@
 }
 
 .pc_pageNumberInput {
-	float: left;
+  float: left;
   width: 2em;
 }
 
 .pc_pageTotal {
-	float: left;
-	padding: 1px 0px 0px 2px;
+  float: left;
+  padding: 1px 0px 0px 2px;
```

Comment:
Missing new line at end of file.

![IntelliJ Ensure New Line At EOF](https://github.com/pentaho/pentaho-platform-plugin-common-ui/assets/284983/eda16a0a-ea33-45bb-8bac-e9c2bc932b99)